tests/issue/issue.cpp

```diff
@@ -0,0 +1,501 @@
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::IMPORT_UNITS_NAME);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::IMPORT_UNITS_REF);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MAP_VARIABLES_UNIQUE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MAP_VARIABLES_VARIABLE1);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MAP_VARIABLES_VARIABLE2);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MODEL_CHILD);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MODEL_ELEMENT);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MODEL_MORE_THAN_ONE_ENCAPSULATION);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::MODEL_NAME);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::RESET_CHILD);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::RESET_VARIABLE_REFERENCE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::RESET_TEST_VARIABLE_REFERENCE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::RESET_ORDER);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNITS_CHILD);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNITS_NAME);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNITS_NAME_UNIQUE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNITS_STANDARD);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_OPTIONAL_ATTRIBUTE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_CIRCULAR_REF);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_DIGRAPH);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_EXPONENT);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_MULTIPLIER);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_PREFIX);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::UNIT_UNITS_REF);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::VARIABLE_INITIAL_VALUE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::VARIABLE_INTERFACE);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::VARIABLE_NAME);
+    ++count;
+    testReferenceRule(e);
+    e->setRule(libcellml::ReferenceRule::VARIABLE_UNITS);
+    ++count;
+    testReferenceRule(e);
+    EXPECT_EQ(size_t(51), count);
+}
+
+TEST(Issue, createModelWarning)
```

Comment:
The below tests being new, it might be nice to be consistent with line spacing? I.e. remove empty lines in the tests below including this one.

Reply:
Done ... though personally I think it makes it harder to read ...

Reply:
I like empty lines here and there, but this is not the style used in our code. I appreciate that those are only tests, but I would still like to see some consistency, if possible.